foreshadow/tests/test_core/test_column_sharer.py

```diff
@@ -1,7 +1,7 @@
 import pandas as pd
 import pytest
 
-from foreshadow.column_sharer import ColumnSharer
+from foreshadow.core.column_sharer import ColumnSharer
```

Comment:
I must have missed this on the first run through, but just wanted suggest that the imports be moved within the tests so that if there is an import error it can be debugged at "test" time rather than "collect" time.